lib/Driver/ToolChains.cpp

```diff
@@ -1471,6 +1519,222 @@ toolchains::Darwin::constructInvocation(const LinkJobAction &job,
   return II;
 }
 
+ToolChain::InvocationInfo
+toolchains::Windows::constructInvocation(const InterpretJobAction &job,
+                                             const JobContext &context) const {
+  InvocationInfo II = ToolChain::constructInvocation(job, context);
```

Comment:
No need for this now that you're inheriting directly from ToolChain.

Reply:
I see this is vestigial because @compnerd had you remove the LD_LIBRARY_PATH trick. Unfortunately we'll need to come up with something similar for Windows if we want the interpreter to work with -L. We're probably not doing the right thing on Windows anyway, though (`dlopen`).